examples/predictors/linear/Example-Train-Linear-Rollout-Model.ipynb

```diff
@@ -1,12 +1,10 @@
 {
```

Comment:
Pretty much same story for the example notebooks:
- fixed Pandas warnings
- added NPI H6_Facial Coverings
- and used the validation file that contains historical npis.